src/EdFi.AnalyticsMiddleTier.Tests/TestCases/StudentAssessmentFact/0E886CBB_AB8F_ABD3_747F_C455C5141145_2009_04_01_100041249_should_have_PerformanceResult_empty.xml

```diff
@@ -9,6 +9,5 @@
         WHERE StudentAssessmentKey = '0E886CBB-AB8F-ABD3-747F-C455C5141145-uri://edfi.org/Assessment-0E886CBB-AB8F-ABD3-747F-C455C5141145_2009-04-01-100041249';
     </Query>
     <Result>
-        <PerformanceResult></PerformanceResult>
```

Comment:
I think that what you are trying to say here is that it shouldn't return any results. I wouldn't say it is incorrect, but in other  cases we do something like 
`SELECT COUNT(1) AS CountValue`
And we have a CountResult class to map the result. You can keep it like this if you want but you would need to change the name of the test, so instead of _should_have_performanceresult_empty, it says that it returns no results.

Reply:
resolved, please review it again.